src/test/integration/ProjectsClient.test.ts

```diff
@@ -26,6 +26,28 @@ describe("ProjectsClient", () => {
     chai.expect(projectList).to.not.be.empty;
   });
 
+  it("should get project details from projectId", async () => {
+    const projectDetails: Project[] = await projectsAccessClient.getAll(accessToken,
+      {
+        pagination: {
+          top: 1,
+        },
+      });
+
+    const mappedFullProjectList: Project[] = projectDetails.map((project) => {
+      return {
+        id: project.id,
+        name: project.name,
+        code: project.code,
+      };
+    });
+
+    const projectInfo: Project = await projectsAccessClient.getByProjectId(accessToken, mappedFullProjectList[0].id);
+
+    chai.expect(projectInfo.name).to.not.be.empty;
```

Comment:
Instead of an empty check, you want to validate that the returned project has the same id, name, and code as the project from projectDetails. By checking if they are not empty you are validating that _a_ project was returned but not validating that _the same_ project was returned.